src/js/backend/BlockchainLink.js

```diff
@@ -331,9 +345,10 @@ export const initBlockchain = async (
             postMessage,
             debug: DataManager.getSettings('debug'),
             proxy: DataManager.getSettings('proxy'),
-            onionDomains: DataManager.getSettings('useOnionLinks')
-                ? DataManager.getConfig().onionDomains
-                : undefined,
+            onionDomains:
+                DataManager.getSettings('useOnionLinks') && !customBackends[coinInfo.shortcut]
```

Comment:
Was it necessary to remove the .onion translation for custom backends? I don't think it's a problem either to have it or not to have it there; I'm just curious.

Reply:
not a problem, it has no effect on suite.  I think we talked about it earlier that it may be confusing that you explicitly requested some custom address and get connected to different one